protocol/x/clob/memclob/memclob.go

```diff
@@ -964,10 +967,17 @@ func (m *MemClobPriceTimePriority) ReplayOperations(
 		// Matches are a no-op.
 		case *types.InternalOperation_Match:
 		case *types.InternalOperation_OrderRemoval:
+			orderId := operation.GetOrderRemoval().OrderId
+
+			// Prevent double placement caused by PreexistingStatefulOrder and Order Removal
+			// both existing in local operations.
+			if _, found := placedStatefulOrderIds[orderId]; found {
+				continue
+			}
```

Comment:
For my understanding - it's valid to have [preexisting placement, removal] (we don't error log here), but not valid to have [removal, preexisting placement], right?